Add personal task manager

MananSoni42 opened the pull request:
Make changes that were requested

Pro-Panda commented on To-do-list/manan/to_do_list_tasks.py:
```diff
@@ -0,0 +1,278 @@
+        os.system(f'/usr/bin/gsettings set org.gnome.desktop.background picture-uri {path}')
+        return path
+
+    # sets up the tasks on the current wallpaper
```
Use docstrings to explain what a function does, not comments

Pro-Panda commented on To-do-list/manan/to_do_list_tasks.py:
```diff
@@ -0,0 +1,278 @@
+
+            self.audio = d["audio"]
+            if not os.path.exists(self.audio):
+                print(f'File {self.audio} no longer Exists! Switching to default reminder Tone')
```
Don't use `f'<string>'`. It's not a popular PEP style and is not supported by all versions. Either use`format()` or string concatenation

MananSoni42 commented:
Changed to f strings to normal strings ( used .format() )
added docstrings at relevant places
created function menu() called in to_do_list.py in main

Pro-Panda commented:
LGTM ?
